terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/codelist/i18n/SimpleI18nCodeList.java

```diff
@@ -214,6 +225,12 @@
      */
     Table<Locale, String, String> codeListTable;
 
+    /**
+     * the default locale as fallback.
+     * @since 5.5.0
```

Comment:
- [ ] Required to change to 5.5.1.

Reply:
Modified by 2ae21b0c0e6b388b078d3fdfc53ccaed0c2838d4

Reply:
pending

Reply:
Pending for project opinion.

Reply:
I was instructed to avoid destructive changes from @hirakuriy .
Therefore, this change is not backported, it changes to `5.5.1`.

ebc06ac288701a6248b4e65bdad89ab5fe863c99